backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/service/JwtService.java

```diff
@@ -0,0 +1,93 @@
+package com.chinguv45tier3team53.tripplannerrest.service;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class JwtService {
+    @Value("${secret-key}")
+    private String secretKey;
```

Comment:
is it possible to use asymmetric keys instead?

Reply:
we might be able to use them after we reach mvp